reader/components/outline-sidebar/OutlineSidebar.scss

```diff
@@ -15,35 +15,48 @@ $expand-icon: url("assets/expand.svg");
     list-style-type: none;
     padding: 0 0 0 1.5rem;
   }
-}
 
-.outline-button {
-  position: relative;
-  top: 0.35rem;
-  margin: -0.5rem 0 0 -1.5rem;
-  padding: 0;
-  width: 1.5rem;
-  height: 1.5rem;
-  background: none;
-  border: none;
-
-  &::before {
-    content: $expand-icon;
-    position: absolute;
-    top: 0;
-    left: 0;
-    display: block;
-    width: 100%;
-    height: 100%;
-    background: $expand-icon no-repeat center / contain;
-    transition: transform 150ms;
-
-    [aria-expanded="true"] > & {
-      transform: none;
+  .outline-button {
+    position: relative;
+    top: 0.35rem;
+    margin: -0.5rem 0 0 -1.5rem;
+    padding: 0;
+    width: 1.5rem;
+    height: 1.5rem;
+    background: none;
+    border: none;
+
+    &::before {
+      content: $expand-icon;
+      position: absolute;
+      top: 0;
+      left: 0;
+      display: block;
+      width: 100%;
+      height: 100%;
+      background: $expand-icon no-repeat center / contain;
+      transition: transform 150ms;
+
+      [aria-expanded="true"] > & {
+        transform: none;
+      }
+
+      [aria-expanded="false"] > & {
+        transform: rotate(-90deg);
+      }
     }
+  }
+
+  .missing-outline {
+    padding: 0.75rem;
+    border-style: dashed;
+    border-color: #bfbfbf;
```

Comment:
Try to use `$gray-*` accent everywhere. This is why we have configuration files.

Reply:
https://github.com/oacore/reader/pull/106/commits/a8d706477d649c2151095655eeaaded63666f0a7